workers/ohsome_quality_analyst/geodatabase/client.py

```diff
@@ -188,19 +142,20 @@ async def get_area_of_bpolys(bpolys: Dict):
                     )
             ) / (1000*1000) as area_sqkm
         """
-    polygon = json.dumps(bpolys["features"][0]["geometry"])
+    geom = json.dumps(bpolys["features"][0]["geometry"])
     async with get_connection() as conn:
-        result = await conn.fetchrow(query, polygon)
+        result = await conn.fetchrow(query, geom)
     return result["area_sqkm"]
 
 
 async def get_bpolys_from_db(
-    dataset: str, feature_id: int
+    dataset: str, feature_id: Union[int, str], fid_field: str
 ) -> geojson.FeatureCollection:
-    """Get geometry and properties from geo database as a geojson feature collection."""
-    logging.info("Get bpolys geometry")
-    # TODO: adjust this for other input tables
+    """Get bounding polygon from the Geodatabase as a GeoJSON Feature Collection."""
```

Comment:
Either `FeatureCollection` is meant to be as proper name and it's written together, or it's written in small letters :)
```suggestion
    """Get bounding polygon from the geodatabase as a GeoJSON FeatureCollection."""
```